Reference/google/protobuf/unittest_custom_options.pb.swift

```diff
@@ -1013,7 +1093,56 @@ public struct ProtobufUnittest_ComplexOptionType3: ProtobufGeneratedMessage {
     "complexoptiontype5": 2,
   ]}
 
-  var unknown = ProtobufUnknownStorage()
+  private class _StorageClass {
```

Comment:
Why is this now generating a storage class?  This seems like unnecessary code bloat.